builder/hcloud/step_create_server.go

```diff
@@ -87,12 +87,33 @@ func (s *stepCreateServer) Run(ctx context.Context, state multistep.StateBag) mu
 	}
 
 	if c.RescueMode != "" {
-		if err := setRescue(context.TODO(), client, serverCreateResult.Server, c.RescueMode, sshKeys); err != nil {
+		ui.Say("Enabling Rescue Mode...")
+		rootPassword, err := setRescue(context.TODO(), client, serverCreateResult.Server, c.RescueMode, sshKeys)
```

Comment:
there is a context available to use in the scope, it's passed into `Run`.
I advise using it here so that a user can cancel operations with a ctrl-c

```suggestion
		rootPassword, err := setRescue(ctx, client, serverCreateResult.Server, c.RescueMode, sshKeys)
```


Optional out of scope: we can also replace all `context.TODO()` usages with ctx.

Reply:
Done with 6339343 :)

Reply:
Nice one 🙂  !